packages/data-context/src/actions/ElectronActions.ts

```diff
@@ -104,4 +106,14 @@ export class ElectronActions {
       return obj.filePath || null
     })
   }
+
+  showSystemNotification (title: string, body: string) {
+    const notification = this.ctx.electronApi.createNotification(title, body, !this.isMac ? getPathToIcon('icon_64x64.png') : undefined)
+
+    notification.on('click', async () => {
+      await this.ctx.actions.browser.focusActiveBrowserWindow()
+    })
```

Comment:
Should we put this behind a param? Might there be situations where we want to generate notifications when in "run" mode and there isn't a browser window to focus?

Reply:
Yeah we should definitely allow the caller to pass a click handler